index.html

```diff
@@ -215,6 +215,6 @@ <h3>Mach andere schlau!</h3>
         <button class="footer__button" id="buttonProfile"><img src="img/account_circle_black_24dp.svg" alt="Profile"/></button>
       </nav>
     </footer>
-    <script src="javascript.js" defer></script>
+    <script src="javastructure.js" defer></script>
```

Comment:
Please add this to the `head` tag and also check the spelling! Your file starts with a capital letter, this might get you into trouble.